include/aspect/material_model/steinberger.h

```diff
@@ -52,6 +52,10 @@ namespace aspect
     class Steinberger: public MaterialModel::Interface<dim>, public ::aspect::SimulatorAccess<dim>
     {
       public:
+        /**
+         * Virtual destructor.
+         */
+        virtual ~Steinberger();
```

Comment:
That doesn't hurt, but also doesn't help because the destructor is already virtual because the one in the base class is virtual.